site/docs/4.1/components/pagination.md

```diff
@@ -108,8 +106,11 @@ Fancy larger or smaller pagination? Add `.pagination-lg` or `.pagination-sm` for
 {% capture example %}
 <nav aria-label="...">
   <ul class="pagination pagination-lg">
-    <li class="page-item disabled">
-      <a class="page-link" href="#" tabindex="-1">1</a>
+    <li class="page-item active" aria-current="page">
```

Comment:
@patrickhlauke: is this change and the change in line 125 right? Now the links aren't disabled anymore, so the cursor still changes to pointer

Reply:
Well yeah, but if you hover the cursor is pointer.

Reply:
On https://getbootstrap.com/docs/4.1/components/pagination/#disabled-and-active-states , the cursor is still the pointer for the active state too.

Reply:
True, thanks.